src/components/ShoppingList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../lib/firebase';
+
+const ShoppingList = () => {
+  const [term, setTerm] = useState('');
```

Comment:
Naming is hard. Looking at this `useState` hook by name, it's not clear what `term` indicates. Though the app is young and we only have one item in the component state we should start by ensuring names are descriptive.

Reply:
Good point! updated it to be [groceryItem, setGroceryItem]